### PR TITLE
[50989] iOS: Adjusted country questionnaire page to fit Norwegian design

### PR DESCRIPTION
**Pull request by jonasofrich**
IMG_0178

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19.iOS/Views/AuthenticationFlow/QuestionnaireCountries/CountryTableCell.cs [hunk 1]
```diff
@@ -24,8 +24,8 @@ public CountryTableCell (IntPtr handle) : base (handle)
 
         public void UpdateCell(CountryDetailsViewModel country)
         {
-            StyleUtil.InitLabelWithSpacing(CountryNameLabel, StyleUtil.FontType.FontRegular, country.Name, 1.14, 16, 28);
-            CheckboxImage.Image = (country.Checked ? UIImage.FromBundle("CheckboxChecked") : UIImage.FromBundle("CheckboxUnchecked"));
```
Comment (nc-kano) [inline NDB.Covid19/NDB.Covid19.iOS/Views/AuthenticationFlow/QuestionnaireCountries/CountryTableCell.cs:28]: Is CheckboxUnchecked already removed from assets? if not and it is not used anywhere else can you also remove it?

**Review by nc-kano — CHANGES_REQUESTED** [review]

File: NDB.Covid19/NDB.Covid19.iOS/Views/AuthenticationFlow/QuestionnaireCountries/QuestionnaireCountriesViewController.cs [hunk 2]
```diff
@@ -51,12 +51,20 @@ public override void ViewWillDisappear(bool animated)
...
+            StyleUtil.InitButtonStyling(NextBtn, QuestionnaireCountriesViewModel.COUNTRY_QUESTIONAIRE_BUTTON_TEXT);
+            SetButtonViewStyle();
+        }
+
+        void SetButtonViewStyle()
+        {
+            ButtonView.BackgroundColor = ColorHelper.DEFAULT_BACKGROUND_COLOR;
+            ButtonView.Layer.MasksToBounds = false;
+            ButtonView.Layer.ShadowOffset = new CoreGraphics.CGSize(0, 5);
+            ButtonView.Layer.ShadowRadius = 5;
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19.iOS/Views/AuthenticationFlow/QuestionnaireCountries/QuestionnaireCountriesViewController.cs:66]: Don't we have an already specified size of this radius in a shared file for all iOS files?

**Review by Frederik-S-N-dk — CHANGES_REQUESTED** [review]

File: NDB.Covid19/NDB.Covid19.iOS/Views/AuthenticationFlow/QuestionnaireCountries/CountryTableCell.cs (hunk 1 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19.iOS/Views/AuthenticationFlow/QuestionnaireCountries/CountryTableCell.cs:28]: Done

File: NDB.Covid19/NDB.Covid19.iOS/Views/AuthenticationFlow/QuestionnaireCountries/QuestionnaireCountriesViewController.cs (hunk 2 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19.iOS/Views/AuthenticationFlow/QuestionnaireCountries/QuestionnaireCountriesViewController.cs:66]: No, this is for the shadow on the view containing the button, and not the button itself.

**Review by nc-kano — APPROVED** [review]

**Pull request merged**